src/plugins/flexsearch.js

```diff
@@ -27,18 +27,21 @@ function formatSearch({ node, root }) {
     return {
       title: `${parentHeading} ${nodeString}`,
       anchor: nodeString,
+      pageContext: [ parentHeading, nodeString ],
     };
   case 'Errors':
     const headingNode = stripBadge(toString(findBefore(root, findBefore(root, node, 'heading'), 'heading')));
     return {
       title: `${headingNode} ${nodeString}`,
```

Comment:
@alexBinns Suggestion: Remove the `Attributes` and `Errors` from the search results. They are being duplicated in the context and it causes long context descriptions and truncation which doesn't present well

![image](https://github.com/centrapay/centrapay-docs/assets/46909693/157f3b34-f49e-4e2e-8e91-0d88587daf97)